### PR TITLE
Nc news 10

### PR DESCRIPTION
**Pull request by hjaox**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -218,6 +217,28 @@ describe('DELETE `/api/comments/:comment_id`',() => {
...
+
+        return request(app)
+        .get('/api/users')
+        .then(({body: {allUsers}}) => {
+            allUsers.forEach(user => {
+                expect(user).toMatchObject(expectedObject);
+            })
+        })
+    })
+})
```
Comment (Hard-Luck) [inline __tests__/app.test.js:242]: I would have this on a key of users

File: __tests__/app.test.js [hunk 2]
```diff
@@ -218,6 +217,28 @@ describe('DELETE `/api/comments/:comment_id`',() => {
...
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String)
+        }
+
+        return request(app)
+        .get('/api/users')
+        .then(({body: {allUsers}}) => {
+            allUsers.forEach(user => {
+                expect(user).toMatchObject(expectedObject);
```
Comment (Hard-Luck) [inline __tests__/app.test.js:238]: Would this test fail or pass if your users array was empty?

**Review by Hard-Luck — CHANGES_REQUESTED** [review]
You just need to add a little more to the test to ensure you are getting some users back, look at the test data and see how many you expect. Then you're good to go.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -218,6 +217,28 @@ describe('DELETE `/api/comments/:comment_id`',() => {
...
+    test('200: returns an array of objects with certain properties', () => {
+        const expectedObject = {
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String)
+        }
+
+        return request(app)
+        .get('/api/users')
+        .then(({body: {users}}) => {
```
Comment (B7Bprog) [inline __tests__/app.test.js:235]: Please check the length of the users array to avoid a false positive test.

File: __tests__/app.test.js [hunk 4]
```diff
@@ -333,4 +354,25 @@ describe('Error handling tests', () => {
             })
         })
     })
+    describe('GET `/api/users` errors', () => {
```
Comment (B7Bprog) [inline __tests__/app.test.js:358]: This issue, where the tables are missing is more like a server side issue, rather than a client side one.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Hector, please see my comments and when it's done, you can merge. 👍

**Pull request merged**